public/locales/en/translation.json

```diff
@@ -61,6 +61,7 @@
     "ellipse": "Ellipse",
     "arrow": "Arrow",
     "line": "Line",
-    "text": "Text"
+    "text": "Text",
+    "lock": "Lock the currently selected element"
```

Comment:
open for ideas..

Reply:
I think that suggests it'll lock the element in place (the way you lock layers in editors). I'd use something like "Keep selected tool active". Dunno, doesn't sound good, either.

Reply:
"Keep selected tool active after drawing"?